### PR TITLE
Remove streaming max request size

### PR DESCRIPTION
**Pull request by JamesNK**
Addresses https://github.com/grpc/grpc-dotnet/issues/401

### CODE REVIEW

File: src/Grpc.AspNetCore.Server/Internal/CallHandlers/ServerCallHandlerBase.cs [hunk 1]
```diff
@@ -103,13 +104,34 @@ static async Task AwaitHandleCall(HttpContextServerCallContext serverCallContext
...
+            var maxRequestBodySizeFeature = httpContext.Features.Get<IHttpMaxRequestBodySizeFeature>();
+            if (maxRequestBodySizeFeature != null)
+            {
+                if (!maxRequestBodySizeFeature.IsReadOnly)
+                {
+                    maxRequestBodySizeFeature.MaxRequestBodySize = null;
+                }
+                else
+                {
+                    httpContext.Features.Set<IHttpMaxRequestBodySizeFeature>(UnlimitedHttpMaxRequestBodySizeFeature.Instance);
```
Comment (davidfowl) [inline src/Grpc.AspNetCore.Server/Internal/CallHandlers/ServerCallHandlerBase.cs]: You don't need to do this.

Comment (JamesNK) [reply src/Grpc.AspNetCore.Server/Internal/CallHandlers/ServerCallHandlerBase.cs]: Why not? Won't MaxRequestBodySize throw an error is IsReadOnly is true.

Comment (JamesNK) [reply src/Grpc.AspNetCore.Server/Internal/CallHandlers/ServerCallHandlerBase.cs]: I re-read the IsReadOnly docs. Does it mean that is too late in the request for changing the body size to have any effect?

Comment (davidfowl) [reply src/Grpc.AspNetCore.Server/Internal/CallHandlers/ServerCallHandlerBase.cs]: > Does it mean that is too late in the request for changing the body size to have any effect?

Right.

**Comment by davidfowl** [comment]
Nice!

**Pull request merged**